yt_dlp/extractor/tiktok.py

```diff
@@ -431,13 +441,32 @@ def _parse_aweme_video_web(self, aweme_detail, webpage_url, video_id):
 
         for download_url in traverse_obj(video_info, (('downloadAddr', ('download', 'url')), {url_or_none})):
             formats.append({
+                **watermark_info(download_url),
                 'format_id': 'download',
                 'url': self._proto_relative_url(download_url),
                 'ext': 'mp4',
                 'width': width,
                 'height': height,
```

Comment:
I suggest removing these 2. `width` and `height` are the size of playAddr. The video of downloadAddr may has different resolution which size is not provided in `video_info`